contracts/core/OTCWrapper.sol

```diff
@@ -0,0 +1,690 @@
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ */
+pragma solidity 0.8.10;
+
```

Comment:
are we going to fix the upper bound values / spot shock factor?

Reply:
Opyn margin system: is an absolute value that comes from those formulas in the picture
Our system: notional * Initial margin %

They enter in conflict in eg. Our system says "x" amount of initial margin but Opyn systems says it's higher - in which case it doesn't allow the vault to open

What setting upper bound values / spot shock factor does is reduce "Opyn's margin required" to nearly zero so that our margin system is the active restriction

Reply:
makes sense. we can bring this up in audit as well